torii/platform/resources/display.py

```diff
@@ -33,6 +34,37 @@ def Display7SegResource(
 		ios.append(attrs)
 	return Resource.family(name_or_number, number, default_name = 'display_7seg', ios = ios)
 
+def HDMIResource(
+	name_or_number: str | int, number: int | None = None, *,
+	clk_p: str, clk_n: str, d_p: str, d_n: str, scl: str, sda: str,
+	hpd: str | None = None, cec: str | None = None,
+	bidir: bool = True, invert: bool = False,
```

Comment:
`invert` + `bidir` seems.. strange - suggest instead `dir`, which can have `i`, `o`, or `io` values, defining whether the port is output-only, input-only or able to be either.